from gpt3_assistant.exceptions.listener_error import ListenerError


class NoInputListenerError(ListenerError):
    pass
